draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -717,6 +737,175 @@ Steps:
 23. Output (x, y)
 ~~~
 
+### Boneh-Franklin Method {#supersingular}
+
+The map2curve_bf(alpha) implements the Boneh-Franklin method {{BF01}} which
+covers the case of supersingular curves E: y^2=x^3+B.
+
+**Preconditions**
+
+This algorithm works for any Weierstrass curve over F_{q} such that A=0 and
+q=2 mod 3.
+
+**Examples**
+
+- y^2=x^3+1
+
+**Algorithm**: map2curve_bf
+
+Input:
+
+ - alpha: an octet string to be hashed.
```

Comment:
nit: here we use `alpha: an...` and below we use `alpha - an...`. Can we be consistent?

Reply:
Done